### DISCUSSION TITLE
Why is the "Issues" section of https://github.com/integrations/slack/ repository completely dead?

### DISCUSSION THREAD

**Discussion by adrian-rybski** [community/community#57233] [Repositories]
### Select Topic Area

Bug

### Body

Hi,

I was wondering why the "Issues" section of the abovementioned repository completely dead? I can see that a lot of people is trying to find a solution to their issues but their questions are left unanswered (including mine). There are over 450 open issues and, from what it looks like, there's only one person to handle them all but you can forget that you'll get a response, not to mention "help". 
We were trying to find some help with setting up [GitHub + Slack Integration](https://github.com/integrations/slack/). We're unable to get the "Enter code" prompt to appear in our new app, when it works fine in the official GitHub app. Is there someone that had similar issue in the past and know how to fix that? Here's my issue: [1707](https://github.com/integrations/slack/issues/1707)
![image](https://github.com/community/community/assets/135125111/8ab5a2aa-966f-45be-a68e-cda7dd59fe6c)

**Comment by FredericVaugeoisFlo** [comment]
+1. It's the same for the teams integration. There are hundreds of issues as of right now and there even are some PR's fixing a bunch of these highly requested issues that are being refused because "you don't accept contributions". Is there any reason to completely kill these repos?

**Reply by pn-1705** [reply]
Hello #1
